### PR TITLE
Add the landing page

### PR DESCRIPTION
**Pull request by mprahl**
landing

### CODE REVIEW

File: src/app/pipes/nodedisplay.ts [hunk 1]
```diff
@@ -4,17 +4,41 @@ import { PipeTransform, Pipe } from '@angular/core';
...
+        case('distgitcommit'):
+            return 'Commit';
+        case('kojibuild'):
+            return 'Brew Build';
+        case('advisory'):
+            return 'Advisory';
+        case('freshmakerevent'):
+            return 'Freshmaker Event';
+        case('containerbuilds'):
+            return 'Freshmaker Build';
```
Comment (yashvardhannanavati) [inline src/app/pipes/nodedisplay.ts]: Just wondering if we can change this to Container Build

Comment (mprahl) [reply src/app/pipes/nodedisplay.ts]: Yeah, that makes sense.

**Comment by mprahl** [comment]
@yashvardhannanavati fixed!

**Comment by yashvardhannanavati** [comment]
:+1:

**Pull request merged**